src/main.rs

```diff
@@ -310,7 +310,7 @@ fn process_events(
                         println!(
                             "[{}] Execute: {:?} from {:?}",
                             Utc::now().format("%Y-%m-%d %H:%M:%SZ"),
-                            ifft.then,
+                            ifft.then.replace("{{}}",path.as_ref().unwrap().to_str().unwrap()),
```

Comment:
Could you use `expect` as is done on line 596? Or better yet, refactor substitution into a separate function.

Reply:
Good idea, I've added a new method.